custom/zipline/models.py

```diff
@@ -0,0 +1,153 @@
+import jsonfield
+from datetime import datetime
+from django.db import models
+
+
+class EmergencyOrderStatusUpdate(models.Model):
+    STATUS_PENDING = 'PENDING'
+    STATUS_RECEIVED = 'RECEIVED'
+    STATUS_REJECTED = 'REJECTED'
+    STATUS_APPROVED = 'APPROVED'
+    STATUS_CANCELED = 'CANCELED'
+    STATUS_DISPATCHED = 'DISPATCHED'
+    STATUS_DELIVERED = 'DELIVERED'
+    STATUS_CONFIRMED = 'CONFIRMED'
+    STATUS_ERROR = 'ERROR'
+
+    class Meta:
+        index_together = [
+            ['order', 'vehicle_number'],
+        ]
+
+    order = models.ForeignKey('EmergencyOrder')
+
+    # The timestamp in CommCareHQ that the status update was received
+    timestamp = models.DateTimeField()
+
+    # The timestamp in Zipline that the status update was sent; this will be
+    # blank for statuses that are not received asynchronously from Zipline like
+    # STATUS_RECEIVED and STATUS_REJECTED
+    zipline_timestamp = models.DateTimeField(null=True)
+
+    # One of the STATUS_* constants above
+    status = models.CharField(max_length=126)
```

Comment:
i decided to stop using choices since it's not actually enforced anywhere, and every time you include a new choice it gives you a warning that you need to run makemgirations because the model changed